tests/utils.go

```diff
@@ -2640,6 +2662,64 @@ func CheckForTextExpecter(vmi *v1.VirtualMachineInstance, expected []expect.Batc
 	return err
 }
 
+func configureIPv6OnVMI(vmi *v1.VirtualMachineInstance, expecter expect.Expecter, virtClient kubecli.KubevirtClient) error {
```

Comment:
This function is only working on cirros images, afaiu.

The FedoraExpecter - which also calls this function - has a [different prompt](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L2721). As can be seen below, this function 'expects' the prompt to be '\$' which matches the [cirrOS prompt](https://github.com/kubevirt/kubevirt/blob/master/tests/utils.go#L2672).

I think this slipped through because the fedora expecter is used (only?) on VMIs that were configured via cloud init - and as such don't rely on this function.

I guess we need to enable this function to handle multiple prompt types - depending on the image used - or, assume this function is only to be used on cirrOS images - leaving cloudInit as the only way to configure the IPs on fedora VMs.

If we choose the latter approach, I think the function should be renamed, to reflect that it must only be used on cirrOS VMIs.

Reply:
done - enabled the function to handle multiple prompt types